src/main/scala/ai/privado/languageEngine/java/processor/JavaProcessor.scala

```diff
@@ -123,18 +124,40 @@ object JavaProcessor {
     * @return
     */
   def createJavaCpg(processedRules: ConfigAndRules, sourceRepoLocation: String, lang: String): Either[String, Unit] = {
-
     println(s"${Calendar.getInstance().getTime} - Processing source code using ${Languages.JAVASRC} engine")
     if (!config.skipDownloadDependencies)
       println(s"${Calendar.getInstance().getTime} - Downloading dependencies and Parsing source code...")
     else
       println(s"${Calendar.getInstance().getTime} - Parsing source code...")
     cpgconfig = Config(inputPath = sourceRepoLocation, fetchDependencies = !config.skipDownloadDependencies)
     val xtocpg = JavaSrc2Cpg().createCpgWithOverlays(cpgconfig)
+    if (config.showUnresolvedFunctionsReport)
+      reportUnresolvedMethods(xtocpg)
     println(
       s"${TimeMetric.getNewTime()} - Base processing done in \t\t\t\t- ${TimeMetric.setNewTimeToLastAndGetTimeDiff()}"
     )
     processCPG(xtocpg, processedRules, sourceRepoLocation)
   }
 
+  def reportUnresolvedMethods(xtocpg: Try[Cpg]): Unit = {
+    var unresolved = 0
+    var total = 0;
+    xtocpg match {
+      case Success(cpg) => {
+        total = cpg.call.methodFullName.l.length
+        unresolved = cpg.call.methodFullName.filter(_.matches("(?i)(.*)(unresolved)(namespace|signature)?(.*)")).l.length
```

Comment:
Would recommmend to have 2 different things 
`unresolved Namespace` and `unresolved signature` separation is needed